src/imgMap.js

```diff
@@ -369,6 +369,46 @@ import to_the_edge from "./music_img/to_the_edge.png";
 import two_days_ago from "./music_img/two_days_ago.png";
 import wandering from "./music_img/wandering.png";
 import welcome_horizons from "./music_img/welcome_horizons.png";
+import seaweed from "./img/seaweed.png";
```

Comment:
Another note to self to add to the codegen script that compresses all of these images to png8 so they aren't so big. No action needed from you.